packages/framework/src/utils/http.utils.ts

```diff
@@ -22,7 +22,7 @@ export const initApiClient = (apiKey: string, baseURL = 'https://api.novu.co') =
       } else if (checkIsResponseError(resJson)) {
         throw new PlatformError(resJson.statusCode, resJson.error, resJson.message);
       } else {
-        throw new BridgeError('Error processing API request to Novu Cloud from Bridge application.');
+        throw new BridgeError(resJson);
```

Comment:
Propagate the actual error to the API client to stop obfuscating the actual error.